packages/overlays/src/LocalOverlayController.js

```diff
@@ -128,10 +126,15 @@ export class LocalOverlayController {
     this.isShown ? this.hide() : this.show();
   }
 
-  _createOrUpdateOverlay(shown = this._prevShown, data = this._prevData) {
+  async _createOrUpdateOverlay(shown = this._prevShown, data = this._prevData) {
     if (shown) {
       this._contentData = { ...this._contentData, ...data };
 
+      // preload popper before render, so the user does not see a different view between render and popper being applied
+      if (!this.popperModule) {
+        await this.__preloadPopper();
```

Comment:
Isn't it better to lazyload then do a loading on demand?
The first popper on the page will have a huge delay with 6kb to be loaded and processed first

Reply:
Now lazy-loads in constructor ;), should make it more smooth when the user clicks it with shitty 3g network because hopefully it start preloading when the LocalOverlayController was instantiated